samples/snippets/pom.xml

```diff
@@ -46,7 +46,7 @@
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-pubsublite</artifactId>
-      <version>0.1.7</version>
+      <version>0.1.9-SNAPSHOT</version>
```

Comment:
Please don't update this to SNAPSHOT as this will get mirrored in https://cloud.google.com/pubsub/lite/docs/reference/libraries

Reply:
Autosynth doesn't seem to be updating this number? At least it didnt in release 1.8 which is confusing.

Reply:
It may have something to do with most snippets folders usually don't require version updates / they use libraries bom. @chingor13 is that right? See history: https://github.com/googleapis/java-pubsublite/commits/master/samples/snippets/pom.xml How do we overwrite the update rule?